File: C/Exercises/1ª Lista de Exercícios/Exercício06.c
#include <stdio.h>

int main()
{

    float salario, porcentagem, total;

    printf("Digite o seu salário: ");
    scanf("%f", &salario);

    printf("Digite o percentual de reajuste: ");
    scanf("%f", &porcentagem);

    total = salario * (1 + (porcentagem / 100));

    printf("O salário final será: R$%f\n", total);

}

File: Java/1ª Lista de Exercícios (1S)/Exercício06/Exercício06.java
import java.util.Scanner;

public class Exercício06{
	public static void main(String args[]){
		Scanner scan = new Scanner(System.in);
		
		System.out.print("Salário: ");
		float salario = scan.nextFloat();

		System.out.print("Reajuste: ");
		float reajuste = scan.nextFloat();

		float novo_Salario = salario + (salario * reajuste);

		System.out.println("Seu novo salário é igual a R$" + novo_Salario);

		scan.close();
		
	}
}
